### PR TITLE
Fix text on screenshots page

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR improves the text on the screenshots page reg. version 2.7.

### CODE REVIEW

File: src/data/screenshots.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         "content": {
             "textblock": [
                 "Browse through the screenshots to get an overview of the features that Corona-Warn-App offers in the current release (Version 2.7 available since 16.07.2021). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
```
Comment (MikeMcC399) [inline src/data/screenshots.json:12]: The date is wrong! It should be 16.08.2021 (August, not July).

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Date in English text is wrong month.

File: src/data/screenshots.json (hunk 1 above)
Comment (Ein-Tim) [reply src/data/screenshots.json]: True, thanks for the hint! Fixed in [`95ea03d` (#1635)](https://github.com/corona-warn-app/cwa-website/pull/1635/commits/95ea03d5976abc12cbb5dc8cde1e059a629819dd)

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 Excellent! Thanks.

**Pull request merged**